ac_runtime-api/modules/babe.adoc

```diff
@@ -41,10 +41,11 @@ type at genesis will be used. Dynamic slot duration may be supported in the futu
 |The randomness for the genesis epoch
 |32-byte array
 
-|_SecondarySlot_
+|_AllowedSlots_
 |Whether this chain should run with secondary slots and wether they are assigned
-in a round-robin manner or via a second VRF.
-| 8bit enum
+in a round-robin manner or via a second VRF (<<sect-block-production-lottery>>).
+|A varying datatype of the following format:
+stem:[{(0,->,"primary slots"),(1,->,"primary & secondary with plain outputs"),(2,->,"primary & secondary with VRF outputs"):}]
```

Comment:
I am not sure if I agree with the new name and notation. `primary and secondary with plain outputs` sounds like the primary slot changes too. Maybe just leave it as `Secondary Slot`, with zero for none, one for plain and two with VRF.

Reply:
Updated:

![image](https://user-images.githubusercontent.com/42901763/161255916-b18d6282-e8a4-46ae-af2d-0025ac56f04f.png)

Reply:
Could you rename the field as well? It seems to be called secondary slot in most code bases as well. then you can remove the `secondary with` as well.

Reply:
Whoops I kind of missed this comment. Do you still want me to change this? I saw that you already made some changes.

Reply:
Yes, sorry, I resolved this.